tests/trainers/java/files_and_directories.java

```diff
@@ -0,0 +1,170 @@
+/**
+Test to check the Java File module.
+It consists on:
+- Text and binary file read and write
+- File operations (e.g. exists, permissions, rename and delete)
+- Directory operations (e.g. read content, creation)
+*/
+import java.io.*;
+import java.io.IOException;
+import java.util.UUID;
+
+class Test {
+    String textContent = "Content of the text file";
+
+    private void createTextFile(String filepath) throws IOException{
+        FileWriter file = new FileWriter(filepath);
+        file.write(this.textContent);
+        file.close();
+    }
+
+    private void readTextFile(String filepath) throws IOException {
+        FileReader file = null;
+        file = new FileReader(filepath);
+        char [] a = new char[1024];
+        int length = file.read(a);
+        String output = "";
+        for(int i=0; i<length; i++)
+            output = output + a[i];
+        file.close();
+    }
+
+    private void createBinaryFile(String inputFile, String outputFile) throws IOException {
+        InputStream inputStream = null;
+        OutputStream outputStream = null;
+        inputStream = new FileInputStream(inputFile);
+        outputStream = new FileOutputStream(outputFile);
+
+        int byteRead = -1;
+        while ((byteRead = inputStream.read()) != -1) {
+            outputStream.write((byte)byteRead);
+        }
+        
+        inputStream.close();
+        outputStream.close();
+    }
+
+    private void length(String filepath, int minSize) throws IOException, Exception {
```

Comment:
Isn't `IOException` a subclass of `Exception`?

Reply:
Ok, I agree, let's simplify the code